pkg/tsdb/v3iotsdb.go

```diff
@@ -201,6 +199,7 @@ func (a *V3ioAdapter) DeleteDB(configExists bool, force bool, fromTime int64, to
 	// delete the Directory object
 	a.container.Sync.DeleteObject(&v3io.DeleteObjectInput{Path: path})
```

Comment:
`a.container.Sync.DeleteObject` may return an error, please check and at least log it
in line 208, we hide the original error. I see no point in doing so